### PR TITLE
LocalCan 2.3.4 (new cask)

### PR DESCRIPTION
**Pull request by paour**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/l/localcan.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  name "LocalCan"
+  desc "Develop apps with Public URLs and .local domains"
+  homepage "https://www.localcan.com/"
+
+  # Documentation: https://docs.brew.sh/Brew-Livecheck
+  livecheck do
+    url "https://www.localcan.com/changelog"
+    regex(/<a href="#(\d+\.\d+\.\d+)">/)
+    strategy :page_match
+    # skip "temp"
```
Comment (khipp) [inline Casks/l/localcan.rb:21]:
```suggestion
    url "https://www.localcan.com/download"
    regex(/href=.*?LocalCan[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
```

File: Casks/l/localcan.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  app "LocalCan.app"
+
+  # Documentation: https://docs.brew.sh/Cask-Cookbook#stanza-zap
+  zap trash: [
+    "~/Library/Application Support/LocalCan",
+    "~/Library/Caches/com.electron.localcan",
+    "~/Library/Caches/com.electron.localcan.Shiplt",
+    "~/Library/Logs/LocalCan",
+    "~/Library/Preferences/com.electron.localcan.plist",
+  ]
```
Comment (khipp) [inline Casks/l/localcan.rb:36]:
```suggestion
    "~/Library/Saved Application State/com.electron.localcan.savedState",
  ]
```

File: Casks/l/localcan.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  # Documentation: https://docs.brew.sh/Brew-Livecheck
+  livecheck do
+    url "https://www.localcan.com/changelog"
+    regex(/<a href="#(\d+\.\d+\.\d+)">/)
+    strategy :page_match
+    # skip "temp"
+  end
+
+  auto_updates true
```
Comment (khipp) [inline Casks/l/localcan.rb:24]: Is the app able to update itself? I might have missed it, but I did not notice any update check or related settings during my test.

**Review by khipp — COMMENTED** [review]
Thank you for your contribution, @paour! I left a few suggestions below. Please also remove any generated comments.

File: Casks/l/localcan.rb (hunk 3 above)
Comment (paour) [reply Casks/l/localcan.rb:24]: You're right, I also didn't see a menu item to check for updates.

File: Casks/l/localcan.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  name "LocalCan"
+  desc "Develop apps with Public URLs and .local domains"
+  homepage "https://www.localcan.com/"
+
+  livecheck do
+    url "https://www.localcan.com/download"
+    regex(/href=.*?LocalCan[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
+  end
+
+  auto_updates false
```
Comment (khipp) [inline Casks/l/localcan.rb:18]:
```suggestion
```

Thanks, we should remove this line altogether.

**Comment by krehel** [comment]
I'm not sure why one wouldn't use the GitHub page for the downloads?  https://github.com/LocalCan/LocalCanApp

This app doesn't meet the notability requirements.

**Comment by paour** [comment]
The author uses Github for distribution and support only. The [releases page](https://github.com/LocalCan/LocalCanApp/releases) seems to be updated only sporadically, which is why I didn't use it.

As for notability, since it's not an open-source project, I don't know what the metric should be. It does have 283 followers on [ProductHunt](https://www.producthunt.com/products/localcan).

**Comment by bevanjkay** [comment]
> This app doesn't meet the notability requirements.

The GitHub notability check doesn't apply here because GitHub is only used for releasing the software not the source code.

**Review by bevanjkay — APPROVED** [review]

**Review by khipp — APPROVED** [review]

**Pull request merged**